### PR TITLE
Final changes to the xarray tutorial

### PR DESCRIPTION
**Pull request by jbusecke**
Minor additions to the notebook

### CODE REVIEW

**Comment by ocefpaf** [comment]
Do you have commit rights here?

File: Xarray/xarray-oceanhackweek21.ipynb [hunk 1]
```diff
@@ -28,7 +28,7 @@
   },
```
Comment (jbusecke) [inline Xarray/xarray-oceanhackweek21.ipynb:28]: <p>Its just this cell and the one below that are new.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/oceanhackweek/ohw-tutorials/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='70'/>

**Comment by jbusecke** [comment]
> Do you have commit rights here?

I might not understand the question correctly. I did basically just add two cells to the notebook. Does that make sense?

**Comment by ocefpaf** [comment]
Sorry, I was typing from my phone earlier and was not clear. I wanted to know if you could merge this yourself. I'll see about adding all the instructors to this repo. It is not nice to have a single point of failure at this moment and you should be able to self-merge things from your own tutorial.

**Comment by jbusecke** [comment]
Got it. Ill try with the other tut I am working on rn.

**Pull request merged**